vsc/model/vsc_ast.py

```diff
@@ -23,63 +26,63 @@ class Error:
 class Method:
     name: str
     description: Optional[str] = None
-    error: Optional[List[Error]] = field(default_factory=lambda: [])
-    input: Optional[List[Argument]] = field(default_factory=lambda: [])
-    output: Optional[List[Argument]] = field(default_factory=lambda: [])
+    error: List[Error] = field(default_factory=EmptyList)
```

Comment:
Makes sense to factor out the lambda.  Reduced repetition of code (even if very small).

But if possible I think we should keep **Optional** in the type hint, because it more clearly/explicitly shows that this item is optional.

Reply:
I brought back the `Optional[...]` , check in case I have missed some of them.